src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Statistics/BoundaryDataController.cs

```diff
@@ -0,0 +1,95 @@
+using GeoJSON.Net.Feature;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Admin.ViewModels.Statistics;
+using GovUk.Education.ExploreEducationStatistics.Common.Model.Data;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api.Statistics;
+
+[Route("api")]
+[ApiController]
+[Authorize]
+public class BoundaryDataController : ControllerBase
+{
+    private readonly IBoundaryDataService _boundaryDataService;
+    private readonly IBoundaryLevelService _boundaryLevelService;
+
+    public BoundaryDataController(
+        IBoundaryDataService boundaryDataService,
+        IBoundaryLevelService boundaryLevelService)
+    {
+        _boundaryDataService = boundaryDataService;
+        _boundaryLevelService = boundaryLevelService;
+    }
+
+    [HttpGet("boundary-data/levels")]
+    public async Task<ActionResult<List<BoundaryLevelViewModel>>> GetBoundaryLevels()
+    {
+        var levels = await _boundaryLevelService.Get();
+
+        return Ok(levels);
+    }
+
+    [HttpGet("boundary-data/levels/{id}")]
```

Comment:
Will that constraint not cause issues with consumers trying to hit the endpoint? It's not common for IDs to be `long`

Reply:
I'm not 100%, but I assumed that was a hint to the model binder to turn the id included in the route into a long - although it's not really necessary as the model binder can infer the type from the controller method's arguments, but we generally add the type to the route string as well, for whatever reason.

Reply:
It will mean that if the value is not a valid _whatever-the-type-is_, then the route will not be matched and the caller will get a 404, which I think is a good thing.